src/modules/cookieHunter/cookieHunter.helpers.ts

```diff
@@ -0,0 +1,187 @@
+  const channel = await cache.get('cookieHunterChannels', []);
+  if (!channel.length) return;
+
+  const randomChannel = channel[Math.floor(Math.random() * channel.length)];
+  if (!randomChannel) return;
+
+  const channelToSend = await client.channels.fetch(randomChannel);
+
+  if (!channelToSend || !channelToSend.isTextBased()) return;
+  const cookieMessage = await channelToSend.send('**👵 Qui veut des cookies ?**');
+  await cache.set('currentHuntMessageId', cookieMessage.id);
+  await cache.set('cookieHunterDailyCount', {});
+  await cookieMessage.react('🥛');
+  await cookieMessage.react('🍪'); // 1 point for grandma here, she beats everyone who doesn't find her
+
+  setTimeout(() => void dailyHuntEnd(client, cookieMessage), ONE_MINUTE);
+};
+
+const handleMilkReaction = async (
+  reaction: MessageReaction | PartialMessageReaction,
+  user: User | PartialUser,
+  isMilkJokerAlreadyFound: boolean,
+) => {
+  if (isMilkJokerAlreadyFound) {
+    await reaction.message.reply({
+      content: `Il est lent ce lait... 🥛`,
+      options: { ephemeral: true },
+    });
+  } else {
+    await cache.set('milkJokerUserId', user.id);
+    await reaction.message.reply({
+      content: `Premier arrivé, premier servit. Cul sec 🥛 !`,
+      options: { ephemeral: true },
+    });
+  }
+};
+
+const applyMilkJoker = async () => {
+  const milkJokerUserId = await cache.get('milkJokerUserId');
+  if (!milkJokerUserId) return;
+
+  const cookieHunterDailyCount = await cache.get('cookieHunterDailyCount', {});
+  const userDailyCount = cookieHunterDailyCount[milkJokerUserId] || 0;
+  const newDailyCount = { ...cookieHunterDailyCount, [milkJokerUserId]: userDailyCount * 2 };
+  await cache.set('cookieHunterDailyCount', newDailyCount);
+};
+
+const logDailyCount = async (client: Client<true>) => {
+  const dailyLogChannels = await cache.get('cookieHunterDailyLogChannels', []);
+  if (!dailyLogChannels.length) return;
+
+  const currentHuntMessageId = await cache.get('currentHuntMessageId');
+  if (!currentHuntMessageId)
+    throw new Error('Lost the hunt message id before logging the daily count');
+
+  const cookieHunterDailyCount = await cache.get('cookieHunterDailyCount', {});
+  const hunterCount = Object.keys(cookieHunterDailyCount).length - 1; // grandma is not a hunter
+
+  const milkJokerUserId = await cache.get('milkJokerUserId');
+
+  const resume = `**🍪 Résumé de la chasse aux cookies du jour**\n`;
+  const where = `Mamie a servi des cookies dans <#${currentHuntMessageId}>\n`;
+  const baseMessage = `${resume}${where}`;
+
+  const message =
+    hunterCount > 0
+      ? getHuntersFoundGrandmaMessage(baseMessage, cookieHunterDailyCount, milkJokerUserId)
+      : `${baseMessage}**🍪 Personne n'a trouvé Mamie !**\nFaut dire qu'elle se cache bien (et que vous êtes nazes) !`;
+
+  for (const channelId of dailyLogChannels) {
+    const channel = await client.channels.fetch(channelId);
+    if (!channel || !channel.isTextBased()) return;
```

Comment:
shouldn't it be `continue` instead?

probably an eslint rule we should have. otherwise this error happens everytime, myself included

Reply:
good catch thanks